hca2scea-backend/test/golden/arguments.csv

```diff
@@ -0,0 +1,5 @@
+spreadsheet,HCA project uuid,E-HCAD accession,curator initials,technology,experiment type,factor values,public release date,hca last update date,study accession

```

Comment:
I recommend adhering to a consistent column naming convention. snake_case_naming is a common practice for data frame column names.